recipe-box-frontend/src/main/frontend/src/components/RecipeView/RecipeDetails.css

```diff
@@ -0,0 +1,35 @@
+h1{
```

Comment:
Set the styling within the recipe-details class in order to avoid style overriding.  This should be available for all the styled elements within the container